packages/jekyll-yaml/0.1.0/README.md

```diff
@@ -0,0 +1,25 @@
+This package lets you add some yaml-metadata to a jekyll post.
+
+Typing `:newpost` yields the following:
+
+```
+---
+date: 2024-12-05
+author: YourName
+tags:
+category:
+title:
+---
+```
+
+Date is set to the current date and authorname can obviously be adjusted.
+
+Please make sure to add tags like the following and remain consistent:
```

Comment:
I assume this self-evident to Jekyll users? Would it be worth adding a few words about _where_ to add the tags?

Reply:
I may add some more functionality through forms later but I think this basic thing works just fine for now. Good idea, though